libbeat/metric/system/process/process_test.go

```diff
@@ -61,9 +58,54 @@ func TestGetOne(t *testing.T) {
 	err := testConfig.Init()
 	assert.NoError(t, err, "Init")
 
-	procData, err := testConfig.GetOne(os.Getpid())
+	_, _, err = testConfig.Get()
+	assert.NoError(t, err, "GetOne")
+
+	time.Sleep(time.Second * 2)
+
+	procData, _, err := testConfig.Get()
+	assert.NoError(t, err, "GetOne")
+
+	t.Logf("Proc: %s", procData[0].StringToPrint())
+}
+
+func TestFilter(t *testing.T) {
```

Comment:
This was the missing test from the looks of things?

Reply:
Yah, added that.